src/commands/add_command/stake_proposal/transfer_near_tokens_type/mod.rs

```diff
@@ -180,28 +155,60 @@ impl TransferNEARTokensAction {
 
 impl TransferNEARTokensAction {
```

Comment:
I find this whole module screaming for refactoring and renaming (starting from the folder name and ending at the proper use of types and allowance checks)